microsoft-edge/webview2/concepts/multi-profile-support.md

```diff
@@ -149,72 +217,109 @@ HRESULT AppWindow::CreateControllerWithOptions()
 
 ---
 
-<!-- end of tab-set -->
-
 
 <!-- ====================================================================== -->
 ## Access and manipulate the profile
 
-You can get the profile object by accessing the profile property of a WebView2 control.
+You can get the profile object by accessing the `Profile` property of a WebView2 control.
 
+##### [.NET/C#](#tab/dotnetcsharp)
 
-<!-- ------------------------------ -->
+* [CoreWebView2.Profile Property](/dotnet/api/microsoft.web.webview2.core.corewebview2.profile#microsoft-web-webview2-core-corewebview2-profile)
 
-# [C#](#tab/csharp)
+##### [WinRT/C#](#tab/winrtcsharp)
 
-TODO
+* [CoreWebView2.Profile Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2#profile)
 
+##### [Win32/C++](#tab/win32cpp)
 
-<!-- ------------------------------ -->
-
-# [C++](#tab/cpp)
-
-[ICoreWebView2_13::get_Profile Method](/microsoft-edge/webview2/reference/win32/icorewebview2_13#get_profile)
+* [ICoreWebView2_13::Profile property (get)](/microsoft-edge/webview2/reference/win32/icorewebview2_13#get_profile)<!--no put-->
 
 ---
 
-<!-- end of tab-set -->
+After you get the profile object, you can manipulate it.  Use `CoreWebView2Profile` to get profile information and do profile-wide settings and operations.
 
+##### [.NET/C#](#tab/dotnetcsharp)
 
-Then you can manipulate the profile object. The Profile interface allows to get profile information and do profile-wide settings and operations.
+* [CoreWebView2Profile Class](/dotnet/api/microsoft.web.webview2.core.corewebview2profile)
 
+##### [WinRT/C#](#tab/winrtcsharp)
 
-<!-- ------------------------------ -->
+* [CoreWebView2Profile Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2profile)
 
-# [C#](#tab/csharp)
+##### [Win32/C++](#tab/win32cpp)
 
-TODO
+* [ICoreWebView2Profile interface](/microsoft-edge/webview2/reference/win32/icorewebview2profile)
+   * [ICoreWebView2Profile2 interface](/microsoft-edge/webview2/reference/win32/icorewebview2profile2) - Methods to clear browsing data.<!--keep text-->
 
+---
 
-<!-- ------------------------------ -->
 
-# [C++](#tab/cpp)
+<!-- ====================================================================== -->
+## Example of accessing and manipulating the profile
 
-[ICoreWebView2Profile Interface](/microsoft-edge/webview2/reference/win32/icorewebview2profile)
 
-[ICoreWebView2Profile2 Interface](/microsoft-edge/webview2/reference/win32/icorewebview2profile2)
+##### [.NET/C#](#tab/dotnetcsharp)
 
----
+<!-- todo: replace code listing? -->
```

Comment:
2. @leonhsl - Replace the below C# code that's copied from spec?
WinRT section has same code.